code/obj/flock/cage.dm

```diff
@@ -173,4 +173,14 @@
 	else
 		return null // give the standard description
 
+/obj/icecube/flockdrone/attack_hand(mob/user as mob)
+	user.visible_message("<span class='alert'><b>[user]</b> kicks [src]!</span>", "<span class='alert'>You kick [src]!</span>")
+	attack_particle(user, src)
+	takeDamage(2)
+	playsound(src, "sound/impact_sounds/Crystal_Hit_1.ogg", 25, 1)
```

Comment:
INTENT_HARM check?

Reply:
Yeah, I have it implemented that way in flock protection PR in adding the component to cages, figured I could just merge changes from here to there or vice versa.